README.md

```diff
@@ -1,13 +1,17 @@
 # Falconieri
 
-Remote Provisioning Gateway
+Modern IP phones can contact a redirect service provider at boot time
+and discover their PBX address with it.
+
+The Falconieri project is a RPS (Redirect and Provisioning Service) gateway 
```

Comment:
Both the terminology is used as a synonym, even if the "Redirect and Provisioning Service" is more correct.
```suggestion
The Falconieri project is a RPS (Redirect and Provisioning Service / Remote Provisioning Service) gateway 
```

Reply:
"Redirect" is what Falconieri actually helps to do: starting from a MAC and a redirect URL it pushes them to the proper provider. Building a full "remote provisioning" gateway seems out of reach (by now...).